### PR TITLE
Add the number of downloads to the package search page

### PR DESCRIPTION
**Pull request by parkerl**
- Issue #107 

@ericmj is this close to what you were looking for?

### CODE REVIEW

**Comment by parkerl** [comment]
Arggh...was distracted in the middle of preparing this PR and neglected to rebase master. Apologies. Fixed.

File: lib/hex_web/web/router.ex [hunk 1]
```diff
@@ -157,7 +157,11 @@ defmodule HexWeb.Web.Router do
 
     Enum.map(packages, fn package ->
       version = latest_versions[package.id]
-      Map.put(package, :latest_version, version)
+      downloads = PackageDownload.package(package)
```
Comment (ericmj) [inline lib/hex_web/web/router.ex]: This will run a query for every package on the page. It needs to be preloaded for the list of packages.

**Comment by ericmj** [comment]
Except for the preloading this is exactly what I had in mind, thank you!

**Comment by parkerl** [comment]
Cool. You'd like that done right in the template?

**Comment by ericmj** [comment]
It should still be in the router.

**Comment by parkerl** [comment]
Oh got it. Sorry.

**Comment by parkerl** [comment]
@ericmj closer to what you where looking for?

**Comment by ericmj** [comment]
@parkerl Yes, thanks!

**Comment by parkerl** [comment]
@ericmj great! Where can I help next?

**Pull request merged**